### PR TITLE
Automations. Add activateFlowById and activateFlowTriggerById

### PR DESCRIPTION
**Pull request by viktor-liablin**
Automations. Add activateFlowById and activateFlowTriggerById

### CODE REVIEW

File: src/automations/index.js [hunk 1]
```diff
@@ -23,6 +23,21 @@ export default class Automations {
     })
   }
 
+  async activateFlowById(flowId, initialData) {
+    if (!flowId || typeof flowId !== 'string') {
+      throw new Error('The "flowId" argument must be provided and must be a string.')
+    }
+
+    if (initialData !== undefined && !Utils.isObject(initialData)) {
+      throw new Error('The "initialData" argument must be an object with an arbitrary structure.')
```
Comment (stanislav-grin) [inline src/automations/index.js:32]: Just `The "initialData" argument must be an object`, no need to require this object to be an arbitrary structure.

File: src/automations/index.js [hunk 2]
```diff
@@ -42,4 +57,23 @@ export default class Automations {
...
+    if (!flowId || typeof flowId !== 'string') {
+      throw new Error('The "flowId" argument must be provided and must be a string.')
+    }
+
+    if (!triggerId || typeof triggerId !== 'string') {
+      throw new Error('The "triggerId" argument must be provided and must be a string.')
+    }
+
+    if (data !== undefined && !Utils.isObject(data)) {
+      throw new Error('The "data" argument must be an object with an arbitrary structure.')
```
Comment (stanislav-grin) [inline src/automations/index.js:71]: same here

File: src/automations/index.js [hunk 3]
```diff
@@ -42,4 +57,23 @@ export default class Automations {
...
+    }
+
+    return this.app.request.post({
+      url  : `${this.app.urls.automationFlow()}/${ flowId }/trigger/${ triggerId }/activate`,
+      data : data || {},
```
Comment (stanislav-grin) [inline src/automations/index.js:76]: codestyle

File: src/automations/index.js (hunk 2 above)
Comment (viktor-liablin) [reply src/automations/index.js]: \+

File: src/automations/index.js (hunk 3 above)
Comment (viktor-liablin) [reply src/automations/index.js]: \+

File: src/automations/index.js (hunk 1 above)
Comment (viktor-liablin) [reply src/automations/index.js]: \+

**Pull request merged**